### DISCUSSION TITLE
Can I avoid downloading the models and checkpoints every time I restart my PC?

### DISCUSSION THREAD

**Discussion by amircoh44** [lllyasviel/Fooocus#2083] [General]
Hi. Every time I start my PC and want to start generating images, I need to wait fifteen minutes to download 6GB of data added to checkpoints. 

Can I restart the app without downloading every time from the beginning?

**Comment by eddyizm** [comment]
Which ones are you downloading? If you are just using the default or one of the presets it downloads the first time, saves them and doesn't download them again.

My guess is you are doing something funky. Please share details on your work flow, step by step.

**Comment by thiner** [comment]
Fooocus will not download models everytime, that must be something wrong with your local environment. Please provide below information which helps to diagnose.

- Fooocus version
- OS version
- The config file of Fooocus

**Comment by atozcreators** [comment]
I have the same problem it download everytime i run the run.bat file
![IMG_20240924_045553](https://github.com/user-attachments/assets/e2795ca8-1b7e-4842-a8aa-5641730c2044)
